app/common_handlers.py

```diff
@@ -0,0 +1,464 @@
+import aiohttp_jinja2
+import json
+
+from aiohttp.web import HTTPFound, RouteTableDef
+from structlog import get_logger
+from aiohttp_session import get_session
+from aiohttp.client_exceptions import (ClientResponseError)
+
+from . import (ADDRESS_CHECK_MSG,
+               ADDRESS_SELECT_CHECK_MSG,
+               ADDRESS_CHECK_MSG_CY,
+               ADDRESS_SELECT_CHECK_MSG_CY)
+
+from .flash import flash
+from .security import check_permission
+from .utils import View, ProcessPostcode, InvalidDataError, InvalidDataErrorWelsh, FlashMessage, AddressIndex, RHService
+
+logger = get_logger('respondent-home')
+common_routes = RouteTableDef()
+
```

Comment:
Is it possible to put at least some brief docstrings in this file (eg for each class) ?  I would like to know in what way things are "common". I assume things are reused in several contexts.

Reply:
Exactly that - these 5(?) routes can be used in both the start journey and the requests journey (both now use the address lookup functionality), so moved them to a new common routes that work with urls from both journeys.
I'll add comments to that effect.